packages/runner-ct/src/.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": [
+    "plugin:@cypress/dev/react",
+    "plugin:@cypress/dev/tests"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "settings": {
+    "react": {
+      "version": "16.12"
+    }
+  },
+  "rules": {
+    "arrow-body-style": "off",
+    "no-unused-vars": "off",
```

Comment:
I tried this a few times, I cannot find a combo that gets the linter to work for every package.
For now I will just extend an existing eslintrc to avoid the duplication.